### PR TITLE
Fix test failures

### PR DESCRIPTION
**Pull request by pnuu**
This PR fixes couple of test failures.

 - [ ] Closes #xxxx 
 - [x] Tests added  
 - [ ] Tests passed 
 - [ ] Passes ``flake8 trollflow2`` 
 - [ ] Fully documented 
 - [ ] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

File: trollflow2/tests/test_trollflow2.py [hunk 1]
```diff
@@ -1554,7 +1554,8 @@ def test_discard_old_data(self):
             job = {'product_list': None, 'input_mda': {'start_time': dt.datetime(2020, 3, 18)}}
             self.assertIsNone(check_metadata(job))
             get_config_value.return_value = {'start_time': -2e6}
-            self.assertIsNone(check_metadata(job))
+            with self.assertRaises(AbortProcessing):
+                check_metadata(job)
```
Comment (mraspaud) [inline trollflow2/tests/test_trollflow2.py:1558]: I don't think that's the right solution, and I actually already fixed this in https://github.com/pytroll/trollflow2/pull/199/commits/99d4a6b9a6bbad8f04dc7874a56b393c0eea5dbf
(https://github.com/pytroll/trollflow2/pull/199 waited for a positive review to be auto merged, but realise it was only a comment. merging now)

**Review by mraspaud — CHANGES_REQUESTED** [review]
Looks good, but the time test needs to be fixed (see inline comment).

File: trollflow2/tests/test_trollflow2.py (hunk 1 above)
Comment (pnuu) [reply trollflow2/tests/test_trollflow2.py]: Fixed by rebasing with current `main`.

**Review by mraspaud — DISMISSED** [review]
LGTM

**Review by mraspaud — APPROVED** [review]
LGTM

**Pull request merged**